### ISSUE TITLE
Role-based email strategy - remove org members permissions

### ISSUE THREAD

**Issue by Jackseed** [blockframes/blockframes#6753]
Currently some emails are received by org admin or super admin only and other to every members. Current rules are not clear so we should update it and make it clear to users as well. 

- [x] List current rules
- [x] Define new ones with the help of business team
- [x] Add clarifications for users

**Comment by 2amarkina** [comment]
Roles ideas are described in [this document](https://docs.google.com/document/d/1PPv9GS_rG2qCZgprMeMpYv_jD_2sO_TW-Aq6mJSF1Gk/edit)

Some ideas about removing notifications with the color system are[ here](https://docs.google.com/spreadsheets/d/1b1K7YfnquxnfCgGGPxIz5ZNIjr9eK_bUrRJqLvPYGIk/edit#gid=1590038159)

Notifications should be shown to a user in fonction of his role. 

**Left to do:** 
- [ ] Validate the actions per role and general strategy with business team ; 
- [ ] Check with dev team if possible to show different notifications to different roles. 
- [ ] If ok, prepare the wireframes accordingly (Organization level page on the user's level, show different actions per role, check the CRM actions)

**Comment by Jackseed** [comment]
We reviewed your propositions with @cams-goussale and it seems to us that in fact the easiest way to go would be to remove roles and place everyone as member with the power of inviting, removing or accepting/declining other members. 
This would greatly simplify organizations and let the users own their org (vs currently where concierge can do everything and other members can't). 

Regarding emails we should remove the unnecessary (member added or removed) and send all the others to everyone, letting each user decide what they want to receive through notification settings. 

To sum up what we advise is: 
- delete roles
- last member of an org can't be removed (to avoid orphan orgs)
- email are sent to all members by default (but can be changed on notif settings individually)
- we should remove unnecessary emails like member added or removed to org

cc @isseymiyuki & @maltmayer, your opinion is welcome

**Comment by 2amarkina** [comment]
ok, got it ! 

So all the members are able to : 
- Add new members
- Remove members 
- Approve members (if someone wants to join org, via invitations like today)
- Get all notifications

Here are some propositions how this part could be optimized (Organization page > Members tab)

![Company_Members (1).png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/8ec3a69b-97a7-4ff6-8183-0890a647d263)

If we give a right to delete an organization here are two screens how this could be realized: supplementary "delete" button on the organization page and the confirmation modal as this action is really important. 

![Company_Organization.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/bbb065f2-f07b-474c-a544-780546e8c43e)

![Company_Organization (1).png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/f97ff9cf-cb01-4437-82cd-7a9a2c046cfe)

[These wireframes can also be found here](https://www.figma.com/file/DfmsxqCvTtIRTzCFzOOgeb/%5B--Organization-Members-%5D?node-id=6%3A7317)

**Comment by cams-goussale** [comment]
hello! Do we want to give them the possibility to delete orgs?

**Comment by 2amarkina** [comment]
as I can see in [#6753 ](https://github.com/blockframes/blockframes/issues/6574) maybe it's not the priority for the moment, but I guess that it would be important to give this possibility to the users in the future. 

For the moment we can keep this feature for CRM only (already have the CTA).

**Comment by Jackseed** [comment]
Yes I confirm, it's not a priority for now. Same as users, as long as we can do it through CRM (and thus help a user who asks us to do it) we have the minimum needed so let's focus on more prior subjects

**Comment by 2amarkina** [comment]
Can go to UI

**Comment by frenchyrules** [comment]
UI work done [here](https://www.figma.com/file/pjUhRNTj0LOmRSj8Hqa97w/Organization?node-id=0%3A1).
![image](https://user-images.githubusercontent.com/85220905/153601312-b30f6080-1799-44f3-ad69-09b3fcd70469.png)

**Comment by RemcoSimonides** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Design---UI-60ab5f96c64ec50014c46754/issues/blockframes/blockframes/6753?planning-poker) with ZenHub @phpgeekfr

**Comment by phpgeekfr** [comment]
Here is a first overview : 
-  delete roles :  
        - front update :  remove conditional ifs to allow everyone to add member, update the action button to allow member removal
        - update permission model, rules and make a migration  + look at #7475
       => approx 1 week / 1 dev
- last member of an org can't be removed (to avoid orphan orgs)
      - update existing method front / back 
      => 0.5 day / 1 dev
-  email are sent to all members by default (but can be changed on notif settings individually)
     - update emails & notifi that are using `getAdminIds`
     - add possibility to change in settings
   => not sure but this can take some time ( process refactoring to prevent the mail sending)
   => approx 3 days / 1 dev
-  we should remove unnecessary emails like member added or removed to org
      I assume we will be keeping notifications ?
   => 0.5 day / 1 week


**I would say 2,5 weeks 1 dev** I case I missed  something or under rated complexity

**Comment by phpgeekfr** [comment]
some E2E tests may also be impacted https://github.com/blockframes/blockframes/pull/8221

**Comment by RemcoSimonides** [comment]
Ye I do think it should stay in Feature team because it's quite a big issue. But if you're bored for 2,5 weeks you can take it of course ^^ Otherwise I plan it in the Roadmap and see who is available to work on it. You'll be the reviewer, no problem 🤝
